gen/build_deploy/bash.py

```diff
@@ -136,9 +136,9 @@
 
 # check if sort -V works
 function check_sort_capability() {
-    $( echo '1' | sort -V >/dev/null 2>&1 || exit 1 )
+    $( echo '1' | sort -V >/dev/null 2>&1 )
     RC=$?
-    if [[ "$RC" -eq "2" ]]; then
+    if [[ "$RC" -eq "0" ]]; then
```

Comment:
We don't want to collapse all nonzero exit statuses into 1, because not all of them indicate an error. An exit status of 1 doesn't mean the command failed, it means input wasn't sorted.